### PR TITLE
feat: add pkl plugin

### PR DESCRIPTION
**Pull request by chrisvander**
## Summary

Description:

- Tool repo URL: [https://github.com/apple/pkl](https://github.com/apple/pkl)
- Plugin repo URL: [https://github.com/mise-plugins/asdf-pkl](https://github.com/mise-plugins/asdf-pkl)

## Checklist

- [x] Format with `scripts/format.bash`
- [x] Test locally with `scripts/test_plugin.bash --file plugins/<your_new_plugin_name>`
- [x] Your plugin CI tests are green
  - _Tip: use the `plugin_test` action from [asdf-actions](https://github.com/asdf-vm/actions) in your plugin CI_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -549,6 +549,7 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
 | pipelight                     | [kogeletey/asdf-pipelight](https://github.com/kogeletey/asdf-pipelight)                                           |
 | pipx                          | [yozachar/asdf-pipx](https://github.com/yozachar/asdf-pipx)                                                       |
 | pivnet                        | [vmware-tanzu/tanzu-plug-in-for-asdf](https://github.com/vmware-tanzu/tanzu-plug-in-for-asdf)                     |
+| pkl                           | [mise-plugins/asdf-pkl](https://github.com/mise-plugins/asdf-pkl)                     |
```
Comment (jdx) [inline README.md:552]: please sort

Comment (chrisvander) [reply README.md]: The script only formatted it, did I not get the alphabet correct here?

**Review by jdx — APPROVED** [review]

**Review by derekYankie — APPROVED** [review]
I approve. Adding Pkl is a wonderful idea! 👍

**Review by ffluk3 — APPROVED** [review]

**Review by jthegedus — APPROVED** [review]
Thanks

**Pull request merged**